src/recipes/vscode/index.js

```diff
@@ -27,6 +29,26 @@ const getEdgeFunctionsPath = ({ config, repositoryRoot }) =>
 
 const getSettingsPath = (repositoryRoot) => join(repositoryRoot, '.vscode', 'settings.json')
 
+const hasDenoVSCodeExt = async () => {
+  const { stdout: extensions } = await execa('code', ['--list-extensions'])
+  return extensions.split('\n').includes('denoland.vscode-deno')
+}
+
+const getDenoVSCodeExt = async () => {
+  await execa('code', ['--install-extension', 'denoland.vscode-deno']).stdout.pipe(process.stdout)
```

Comment:
Should we also pipe `stderr`?

Reply:
I think we do not even need to manually pipe stdout (or stderr for this matter) as this is the default behavior of `execa`. At least by looking at the options: https://github.com/sindresorhus/execa#stdout-1

So if we remove the `stdout.pipe(...)` part does it still work?

Reply:
In my local testing, if I remove the pipe commands it does not log anything, I would have to save the stdout from the response like const { stdout } = await execa... then console.log(stdout) etc.  Which is a roundabout way of doing the same thing as piping it imo.

Reply:
I think the default behaviour is to capture the output into the return value of the `execa` call. If you also want to pipe it into stdout/stderr, you need to make that explicit.